### ISSUE TITLE
Since `{insight}` offers the same functionality, is `download_data.R` necessary?

### ISSUE THREAD

**Issue by IndrajeetPatil** [easystats/circus#15]
https://github.com/easystats/circus/blob/e222abfef43d90a2d319d36fa627ccf919645bd3/R/download_data.R

**Comment by strengejacke** [comment]
No, not really. We only have this repo as online-storage, it's not intended to install this as a package for users.
